dbt/adapters/athena/session.py

```diff
@@ -41,7 +49,14 @@ class AthenaSparkSessionManager:
     A helper class to manage Athena Spark Sessions.
     """
 
-    def __init__(self, credentials: Any, timeout: int, polling_interval: float, engine_config: Dict[str, int]) -> None:
+    def __init__(
+        self,
+        credentials: Any,
+        timeout: int,
+        polling_interval: float,
+        engine_config: Dict[str, int],
+        relation_name: str | None = None,
```

Comment:
This `| None ` is causing an issue with the unit test suite some how

Reply:
Let me default to hardcoded N/A as this input variable I added is only for logging purpose